canonical-websites/www.canonical.com
# Handle errors with canonicalwebteam.get-feeds

## Done

During my v1 work, Robin landed https://github.com/canonical-websites/www.canonical.com/pull/164 which affected quite a few templates making rebasing or merging onto master very sticky. After quite a few aborted attempts, I found the easiest way was to go through that PR and reimplement the changes in the revised templates.

## QA

`./run serve --watch`

Then go to `/careers`, `/about`, `/partners` and check all the feeds work.

For extra marks, break some feeds by changing the feed URLs in the templates to erroneous ones.

Reload the pages and check you see an error message.
